Pipelines/Pipeline.yml

```diff
@@ -0,0 +1,28 @@
+name: CD-ApiMDeveloperPortal-Build
+
+trigger: none
+
+variables:
+  - name: poolName
+    value: "agentpoolName"
+  - name: location
+    value: West US
+
+jobs:
+
+  # All tasks on APIM Developer portal pipeline
+  - job: Deploy_APIM_Developer_Portal
```

Comment:
`- job: Deploy_APIM_Developer_Portal`
I assume, it's not a deployment, it is migration. Could you please adjust the wording here and everywhere in the file?

Reply:
Updated all places.